deb-packages.txt

```diff
@@ -0,0 +1 @@
+python3-dev python3-venv libjpeg-dev postgresql redis-server libpq-dev libffi-dev virtualenvwrapper
```

Comment:
virtualenvwrapper is not a strict requirement and thus can be listed as additional install in the recommendation section

Reply:
Ok.